server/util/firebaseutil.js

```diff
@@ -13,6 +13,10 @@ function verifyIdTokenAsync(token) {
     return admin.auth().verifyIdToken(token);
 }
 
+function getUserByEmailAsync(email) {
+    return admin.auth().getUserByEmail(email);
+}
+
```

Comment:
We probably also need to modify addCurrentUser from datastore.js so that can't just set fake instance data. That shouldn't be hard because we already call linkInstance - so the code can go in there. We just need to replace setObject with setLocalObjectView (doesn't exist yet), and update the database access rules in psi-product

Reply:
as discussed, will do this later.